### DISCUSSION TITLE
Bezugsquelle ändern

### DISCUSSION THREAD

**Discussion by MrMattn** [marq24/ha-evcc#11] [Q&A]
Hi
wo kann ich ändern, von wo dieses EVCC Addon seine Daten erhält?
Ich hatte EVCC auf Home Assistant und auf dem Proxmox Server in einem Container gleichzeitig laufen fälschlicherweise.
Habe EVCC auf HA deaktiviert doch nun bekomme ich keine Daten mehr vom Addon!?

**Comment by marq24** [comment]
ich kann Deiner Frage nicht so richtig folgen - wenn Du einen neuen EVCC server hast, dann musst Du die aktuelle Konfiguration diese Integration löschen und eine neue (mit der neuen URL des neuen EVCC's) erstellen ?!

**Comment by MrMattn** [comment]
ah ok. ich kann also nirgends sehen, wo die EVCC Integration ihre Daten her bekommt?
Ich kann mich noch dran erinnern, dass man die IP von EVCC im Netzwerk angeben musste, damit die Integration weiß, wo sie ihre Daten her bekommt richtig?

also einmal löschen und neu installieren?

**Comment by marq24** [comment]
also wenn sich diese URL geändert hat - dann "ja" einmal löschen und wieder (mit der neuen URL) hinzufügen...

**Comment by MrMattn** [comment]
wollte in HACS EVCC entfernen. 
doch was tun? wenn ich auf Navigieren klicke, komme ich zu "Einstellungen"-"Integrationen"

![Bildschirmfoto 2024-07-13 um 18 44 37](https://github.com/user-attachments/assets/d9b76e55-be95-4eaa-becd-aa801e0f512e)

**Comment by marq24** [comment]
![image](https://github.com/user-attachments/assets/b9c2b485-72b3-4041-875c-d1fb5fd3ce64)

**Reply by MrMattn** [reply]
ähm das verwirrt mich jetzt. ich hatte das hier doch unter hacs installiert?
weil wenn ich es dort lösche, was ich nun getan habe, kann ich es unter hacs immer noch finden.

**Comment by marq24** [comment]
Du musst nicht die Integration entfernen - sondern "nur" Deine Konfiguration der Integration... das klingt vielleicht für Dich wie das gleiche... ist es aber nicht... mit HACS installirst du den Code der Integration (lokal) - sodass Du die Integration unter den HA-Einstellungen "Geräte und Dienste" hinzufügen kannst... [Du also eine Konfiguration der Integration anlegst]...

Diese config musst Du entfernen/löschen - und anschließend nochmal eine neue hinzufügen...

**Reply by MrMattn** [reply]
oh je wie kompliziert.
hm also egal ob ich jetzt unter (HA / Einstellungen / Integrationen) oder unter (Hacs / Integrationen) suche, nirgends ist mehr was mit evcc zu finden.
also evcc Integration von dir wieder hinzugefügt...Ip eingetragen. alles gut. Er findet 65 Entitäten.
Aber wenn ich auf die für mich wichtigste (Ladeleistung) klicke, dann steht da:
Diese Entität ist deaktiviert.

warum?

**Reply by marq24** [reply]
k.A. - einfach wieder in HA aktivieren?!

**Reply by MrMattn** [reply]
Frage
Muss unter hacs integration was zu finden sein mit evcc?

**Reply by MrMattn** [reply]
![IMG_8851](https://github.com/user-attachments/assets/9b477dbf-d240-48fb-a177-ae7108738f3c)

**Reply by marq24** [reply]
Wenn sich er "Schalter" 'Aktiviert' bei Dir nicht bewegen lässt, dann habe ich auch keine Ahnung, was Du misskonfiguriert haben könntest - bzw. weshalb das bei Deiner Installation nicht so funktioniert - ich kann nur vermuten, dass Du bei dem "dem Repository zu HACS hinzufügen/entfernen/hinzufügen" irgendwas krumm geworden ist... Aber ob wie wie man das fixed - sorry keine Ahnung - liegt m.E. nicht an irgendwas speziellem an der Integration

**Comment by 1977michael** [comment]
Hallo @marq24,

ich überlege, EVCC von meiner QNAP-NAS (Container) auf meinen Linux-Server (Container) anzuziehen. 
Dass dann in HA das bestehende Gerät in der Integration gelöscht und neu hinzugefügt werden muss, ist soweit klar.
Die Frage ist aber, ob die bereitgestellten Entitäten dann wieder die gleichen Namen haben, damit die Helfer wieder ihre Arbeit tun können...

Ich habe eine ganze Reihe an Helfern für Ertrag, Netzbezug und PV-Erzeugung erstellt.
<img width="836" alt="image" src="https://github.com/user-attachments/assets/5e8f8fd4-0acd-4f95-9aba-5d78f5c2c69e" />

<img width="556" alt="image" src="https://github.com/user-attachments/assets/cee506f5-ef44-4d93-be32-d7c6979123de" />

**Reply by marq24** [reply]
Wenn Du die Integartion auf einem neun System erstellst (und die gleichen optionen auwählst ('evcc' hinzufügen JA/NEIN), dann haben auch im neuen System die Sensoren die gleichen Bezeichner

**Reply by 1977michael** [reply]
Mein Plan ist aber nicht HA auf einen neuen Server umzuziehen. HA (einschließlich Deiner EVCC-Integration marq24/ha-evcc) soll weiterhin auf dem RPi laufen.
Der Umzug soll das produktive EVCC-System selbst betreffen.

**Reply by marq24** [reply]
solange die config die gleichen LoadPoints enthält, sollte das auch kein Problem sein...

**Reply by 1977michael** [reply]
Hallo, der Umzug auf den neuen Server hat super funktioniert. 

Gerät in der Integration gelöscht, neu hinzugefügt und alles läuft!

Danke!
